### ISSUE TITLE
Get runes/insignias prices from https://www.presearing.com/runes-and-insignias-pricing when you are in presearing

### ISSUE THREAD

**Issue by bobobaboo** [gwdevhub/GWToolboxpp#1063]
Instead of showing prices from the rune trader

**Comment by AlexMacocian** [comment]
Yeah just checked the html code for it, it's horrible as you said so tbh idk, maybe yeah just block it.

Does Meir manually maintain those prices? Maybe we could ask him to make some page that's just cleartext with the list of prices, if he's anyway maintaining them manually

**Comment by AlexMacocian** [comment]
Spoke with Meir about providing a more app-friendly list of prices for pre. Once they expose that info into their website, I'll take a look at how to parse it and populate prices
